build.py

```diff
@@ -58,15 +58,27 @@ def get_os():
     username, channel, version = get_env_vars()
     reference = "{0}/{1}".format(name, version)
     upload = "https://api.bintray.com/conan/{0}/public-conan".format(username)
+    upload_only_when_stable = True
```

Comment:
this is unnecessary, because conan-package-tools checks both argument and env var using OR conditional:
https://github.com/conan-io/conan-package-tools/blob/master/conan/packager.py#L147

So, you could remove all `upload_only_when_stable` from this file to assume as False by default. Anyone should be able to set as True by env var.

Reply:
Ops, one line below.